ui/admin/app/components/form/storage-bucket/aws/index.hbs

```diff
@@ -453,30 +453,41 @@
   {{/if}}
 
   {{! worker_filter }}
-  <Hds::Form::TextInput::Field
-    name='worker_filter'
-    @isRequired={{true}}
-    @value={{@model.worker_filter}}
-    @isInvalid={{@model.errors.worker_filter}}
-    disabled={{form.disabled}}
-    {{on 'input' (set-from-event @model 'worker_filter')}}
-    as |F|
-  >
-    <F.Label>{{t 'resources.storage-bucket.form.worker_filter.label'}}</F.Label>
-    <F.HelperText>
-      {{t 'resources.storage-bucket.form.worker_filter.help'}}
-      <Hds::Link::Inline @href={{doc-url 'storage-bucket.worker-filter'}}>
-        {{t 'actions.learn-more'}}
-      </Hds::Link::Inline>
-    </F.HelperText>
-    {{#if @model.errors.worker_filter}}
-      <F.Error data-test-worker-filter-error as |E|>
-        {{#each @model.errors.worker_filter as |error|}}
-          <E.Message>{{error.message}}</E.Message>
-        {{/each}}
-      </F.Error>
-    {{/if}}
-  </Hds::Form::TextInput::Field>
+  {{#let (unique-id) (unique-id) as |labelId helpId|}}
+    <Hds::Form::Fieldset
+      aria-labelledby={{labelId}}
+      aria-describedby={{helpId}}
+      class='storage-bucket-worker-filter'
+      @isRequired={{true}}
+      as |F|
+    >
+      <F.Legend id={{labelId}}>{{t
+          'resources.storage-bucket.form.worker_filter.label'
+        }}</F.Legend>
+      <F.HelperText id={{helpId}}>
+        {{t 'resources.storage-bucket.form.worker_filter.help'}}
```

Comment:
nit: @cameronperera I'm wondering if the text translation for this line where it reads `Boolean expression.` could be a clearer, complete sentence?

Reply:
Great feedback. We pull these descriptions from the protobufs. The one for storage buckets can be found [here](https://github.com/hashicorp/boundary/blob/main/internal/proto/controller/api/resources/storagebuckets/v1/storage_bucket.proto#L89) and it actually looks like it was updated. I can update ours as well to match.

Reply:
Thank you! 👍